spec/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require './spec/controllers/api/v1/authenticated_user'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  include_context 'Authenticated User'
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion with unauthorized user' do
```

Comment:
Are you sure the BookSuggestion you are testing here is not authorize? 🤔Because you are including the 'Authenticated User'. Remember the thing I said about autentication in a function instead of a a context 😉

Reply:
U should test here that the BookSuggestion user_id is nil